src/main/scala/tech/cryptonomic/conseil/routes/Tezos.scala

```diff
@@ -68,91 +71,95 @@ object Tezos extends LazyLogging {
   }
 
   val route: Route = pathPrefix(Segment) { network =>
-    get {
-      gatherConseilFilter{ filter =>
-        pathPrefix("blocks") {
-          pathEnd {
-            ApiOperations.fetchBlocks(filter) match {
-              case Success(blocks) => complete(JsonUtil.toJson(blocks))
-              case Failure(e) => failWith(e)
-            }
-          } ~ path("head") {
-            ApiOperations.fetchLatestBlock() match {
-              case Success(block) => complete(JsonUtil.toJson(block))
-              case Failure(e) => failWith(e)
-            }
-          } ~ path(Segment) { blockId =>
-            ApiOperations.fetchBlock(blockId) match {
-              case Success(block) => complete(JsonUtil.toJson(block))
-              case Failure(e) => failWith(e)
-            }
-          }
-        } ~ pathPrefix("accounts") {
-          pathEnd {
-            ApiOperations.fetchAccounts(filter) match {
-              case Success(accounts) => complete(JsonUtil.toJson(accounts))
-              case Failure(e) => failWith(e)
-            }
-          } ~ path(Segment) { accountId =>
-            ApiOperations.fetchAccount(accountId) match {
-              case Success(account) => complete(JsonUtil.toJson(account))
-              case Failure(e) => failWith(e)
-            }
-          }
-        } ~ pathPrefix("operation_groups") {
-          pathEnd {
-            ApiOperations.fetchOperationGroups(filter) match {
-              case Success(operationGroups) => complete(JsonUtil.toJson(operationGroups))
-              case Failure(e) => failWith(e)
-            }
-          } ~ path(Segment) { operationGroupId =>
-            ApiOperations.fetchOperationGroup(operationGroupId) match {
-              case Success(operationGroup) => complete(JsonUtil.toJson(operationGroup))
-              case Failure(e) => failWith(e)
+    validate(conf.hasPath(s"platform.tezos.$network"), s"Network not supported, current options include: 'zeronet'") {
+      get {
+        validate(conf.hasPath(s"platforms.tezos.$network"), s"$network not supported, current options include: 'zeronet'") {
```

Comment:
Is this redundant?

Reply:
Wow, I have no idea how that happened. Definitely redundant